src/components/LongTermAverageGraph/LongTermAverageGraph.js

```diff
@@ -62,6 +50,7 @@ function LongTermAverageGraph({longTermData}) {
                     type: 'scatter',
                     mode: 'lines+markers',
                     marker: {color: 'red'},
+                    hovertemplate: '%{y:.2f}\u00B0C<extra></extra>',
```

Comment:
Same here, can you use whatever units the API call gives you so this graph can support more kinds of data?